docs/docs/platform/step-filter.md

```diff
@@ -0,0 +1,137 @@
+---
+sidebar_position: 9
+---
```

Comment:
I understand this depends on a Webhook so should it be located in position 10 after Webhooks entry?

Reply:
Now sure if they are fully connected. 
@scopsy what do you think?